### ISSUE TITLE
sql_metadata wasn't added to the dependencies

### ISSUE THREAD

**Issue by jefer94** [chrisclark/django-sql-explorer#613]
```
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/jefer/.local/share/virtualenvs/apiv2-bqPrB3_w/lib/python3.11/site-packages/explorer/urls.py", line 9, in <module>
    from .assistant.views import assistant_help
  File "/home/jefer/.local/share/virtualenvs/apiv2-bqPrB3_w/lib/python3.11/site-packages/explorer/assistant/views.py", line 10, in <module>
    from explorer.assistant.utils import (
  File "/home/jefer/.local/share/virtualenvs/apiv2-bqPrB3_w/lib/python3.11/site-packages/explorer/assistant/utils.py", line 4, in <module>
    from sql_metadata import Parser
ModuleNotFoundError: No module named 'sql_metadata'
```

**Comment by jefer94** [comment]
Add the dependency doesn't resolve the issue in GitHub Actions
https://github.com/breatheco-de/apiv2/actions/runs/8842362945/job/24280907243

**Comment by chrisclark** [comment]
this is resolved in #611 which will come out as a release hopefully by end of day today. Also referenced in #610

**Comment by chrisclark** [comment]
Now released:
https://github.com/chrisclark/django-sql-explorer/releases/tag/4.2.0
